forschungsjournal-soziale-bewegungen-fjsb.csl

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Forschungsjournal Soziale Bewegungen FJSB(German)</title>
+    <title-short>FJSB (German)</title-short>
+    <id>http://www.zotero.org/styles/forschungsjournal-soziale-bewegungen-fjsb</id>
+    <link href="http://www.zotero.org/styles/forschungsjournal-soziale-bewegungen-fjsb" rel="self"/>
+    <author>
+      <email>jannis.grimm@fu-berlin.de</email>
+      <name>Jannis Grimm</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="linguistics"/>
```

Comment:
That doesn't seem right? social_science perhaps?

Reply:
That's correct, thanks. Will change it.

Reply:
[8_FJSB Formatstandards_für AutorInnen_08.2019_final.pdf](https://github.com/citation-style-language/styles/files/7492338/8_FJSB.Formatstandards_fur.AutorInnen_08.2019_final.pdf)

This is the file with the style guide (in German), unfortunately, there's no online version